src/browser.js

```diff
@@ -1,67 +1,79 @@
 if (window.ga === undefined) {
-	require('./ga-loader')
+	require('./ga-loader');
 }
 
-var Promise = require('bluebird')
-var TRACKER_NAME = 'resinAnalytics'
+const TRACKER_NAME = 'balenaAnalytics';
```

Comment:
@gelbal @roman-mazur is this ok to change?

Reply:
https://github.com/balena-io-modules/resin-universal-gosquared/pull/3#discussion_r455060239